### PR TITLE
Transfer repo to an org! github.com/divad12 --> github.com/vim-awesome

### PR DESCRIPTION
**Pull request by divad12**
The organization has been set up! http://github.com/vim-awesome

@captbaritone Review please?

Anything else we want to do before pulling the trigger to transfer to the new org? Here's what I can think of:

- [ ] merge this in
- [ ] transfer to vim-awesome org
- [ ] update server's git repo to point to the new one
- [ ] update Gitter chat room?

I think GitHub will automatically redirect the old url to the new one, so that should take care of a lot of potential issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,9 +78,9 @@ repos on GitHub, it is still a useful source of relative usage data.
 
 ## Contributing
 
-Take a look at [some of these issues](https://github.com/divad12/vim-awesome/issues?labels=easyfix&state=open) to get started.
+Take a look at [some of these issues](https://github.com/vim-awesome/vim-awesome/issues?labels=easyfix&state=open) to get started.
 
-Chat with us on [Gitter](https://gitter.im/divad12/vim-awesome)!
+Chat with us on [Gitter](https://gitter.im/vim-awesome/vim-awesome)!
```
Comment (divad12) [inline README.md]: Could this be a potential issue? Not sure how smoothly renaming Gitter rooms is going to be ...

https://github.com/gitterHQ/gitter/issues/153

Comment (captbaritone) [inline README.md]: I'll do a little test. I'll create a throw-away repo, get it running on Gitter, and then attempt to move it to an Org and see what happens.

**Comment by captbaritone** [comment]
It looks like there are a few other GitHub links in `web/static/js/AboutPage.jsx` and `web/static/js/Footer.jsx`. Not a big deal since GitHub should 301 redirect, but since you were updating the Readme, I figured you might want to update those as well.

Also, since each deploy does a fresh clone (and we've updated the clone command) I don't think we'll need step number three.

**Comment by divad12** [comment]
Oops, I missed some. Will update now!

**Comment by divad12** [comment]
Ok, I'm going to go ahead and merge and transfer this now! *fingers crossed*

**Comment by divad12** [comment]
... and deployed! Links now point to github.com/vim-awesome/vim-awesome on the site :)

**Comment by captbaritone** [comment]
:clap: Thanks! It doesn't look like I can setup integrations (Travis, Greenkeeper) on the new repo. Is it possible that I just need different organization permissions?

**Comment by divad12** [comment]
Oops! I just made you an owner. You should be able to do that stuff now!

**Comment by captbaritone** [comment]
@divad12 That did it :wink:

**Pull request merged**